pkgs/os-specific/linux/rtl88xxau-aircrack/default.nix

```diff
@@ -20,14 +20,6 @@ stdenv.mkDerivation rec {
 
   NIX_CFLAGS_COMPILE="-Wno-error=incompatible-pointer-types";
 
-  # Fix build for 5.12 kernels
-  patches = [
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
https://github.com/NixOS/nixpkgs/pull/145808